pages/index.js

```diff
@@ -3,8 +3,46 @@ import Head from 'next/head';
 import styles from './index.module.scss';
 import ApprenticeQualities from '../components/apprentice-qualities/apprentice-qualities';
 import Hero from '../components/hero/hero';
+import { currentApprenticeClassPropTypes, CurrentApprentices } from '../components/current-apprentices/current-apprentices';
 
-const Home = () => (
+export const getStaticProps = async () => ({
+  props: {
+    currentApprenticeClass: {
+      version: '14.0',
+      currentApprentices: [
+        {
+          name: 'Angus Chang',
+          image: '/apprentices/Angus.png',
+          links: [
+            { href: 'personal', text: 'Personal' },
+            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
+            { href: 'https://www.github.com/', text: 'Github' },
+          ],
+        },
+        {
+          name: 'Alice Russel',
+          image: '/apprentices/Alice.png',
+          links: [
+            { href: 'personal', text: 'Personal' },
+            { href: 'https://www.linkedin.com/', text: 'Linkedin' },
+            { href: 'https://www.github.com/', text: 'Github' },
+          ],
+        },
+        {
```

Comment:
Is Alice currently in here twice just to show the layout w/ 3 people? Are you planning to remove this before merging?

Reply:
Yeah, I originally just had 2 cards in the layout and wanted to test it with 3. I removed the duplicate in the latest push.